lms/static/sass/course/layout/_courseware_header.scss

```diff
@@ -45,19 +45,17 @@
         &:hover,
         &:focus {
           color: $blue;
-          border-color: $blue;
+          border-bottom: 4px solid $blue;
         }
 
         &.active {
-          border-bottom: 3px solid $gray-d4;
+          border-bottom: 4px solid $blue;
```

Comment:
The color has no effect on the outcome of accessibility. It was merely a change suggested by Roderick of UX. @lamagnifica FYI